src/hooks/useAuth.tsx

```diff
@@ -3,34 +3,48 @@ import {
   createUserWithEmailAndPassword,
   signOut,
   signInWithEmailAndPassword,
+  onAuthStateChanged,
 } from "firebase/auth";
 import { useState } from "react";
 
 export const useAuth = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [isLogged, setIsLogged] = useState(false);
+
+  const authCheck = () => {
+    onAuthStateChanged(auth, (user) => {
+      if (user) {
+        setIsLogged(true);
+        // console.log("uid: " + user.uid);
+      } else {
+        setIsLogged(false);
+        // console.log("user is not logged");
+      }
+    });
+  };
 
   const register = async () => {
     try {
       await createUserWithEmailAndPassword(auth, email, password);
     } catch (err) {
-      alert(err);
+      console.log(err);
     }
   };
 
   const login = async () => {
     try {
       await signInWithEmailAndPassword(auth, email, password);
     } catch (err) {
-      alert(err);
+      console.log(err);
```

Comment:
I tried to use it to login user and there's error because this function gets empty strings.

Reply:
You might've supplied it with empty string, but it definitely works if all is inputted correctly